### PR TITLE
Pull request to my updated Portfolio Website

### PR DESCRIPTION
**Pull request by braincee**
**_In this pull request, the following requirements were implemented:_**

- I added my recent work projects and about me sections to my portfolio website. 
- I made it mobile responsive which you can use on all mobile devices.
- I used grid and flex layout to make my portfolio website look great.
-  I used media queries to make the website mobile responsive.

- I made some changes to my review response which are : 
    - changed my commit messages to imperative tense
    - changed my button from input tag to button tag.
    - changed the background color of the button and removed the border-radius.
    -  changes the font family of list items in the recent work container to 'Inter', san-serif.
    - added padding to the Recent work title.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,87 +1,183 @@
...
+                   Intuitive Drag & Drop. Up-To Date Information.
+                  </p>
+                  <ul class="skills-4">
+                    <li>CSS</li>
+                    <li>HTML</li>
+                    <li>JavaScript</li>
+                  </ul>
+               <div class="view">
+                  <input class="btn-1" type="button" value="VIEW">
+              </div>
```
Comment (mahbubul14) [inline index.html]: - [ ] This button should be in a link `a` or `button` tag and it should have button interactions (enable, hover, etc.) also. :writing_hand: 

> **Figma design**

e1

File: style.css [hunk 2]
```diff
@@ -145,49 +155,211 @@ li {
...
+
+.btn-1 {
+  width: 100px;
+  height: 50px;
+  padding: 10px 10px;
+  background: #9390ff;
+  color: black;
+  align-self: flex-start;
+  border-radius: 12px;
+}
```
Comment (mahbubul14) [inline style.css]: - [ ] Your button is not matched with the Figma design. You can remove the ` border-radius: 12px;` and change background-color to: `#7D73FC` for matching the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
b1 | b2

File: index.html [hunk 3]
```diff
@@ -1,87 +1,183 @@
...
+                 <p>Add, Prioritize & Organize your Team's Tasks! Be more efficient and successful! Start with. 
+                    A simple To-Do-List, add more Details as you go ! 
+                   Free for up to 50 User! Try it now! Instant Messaging. 
+                   Intuitive Drag & Drop. Up-To Date Information.
+                  </p>
+                  <ul class="skills-4">
+                    <li>CSS</li>
+                    <li>HTML</li>
+                    <li>JavaScript</li>
+                  </ul>
```
Comment (mahbubul14) [inline index.html:111]: - [ ] The font is not matched with the Figma design. Please input the correct font family for this one and the rest of them also.

File: index.html [hunk 4]
```diff
@@ -1,87 +1,183 @@
...
-        </div>
\ No newline at end of file
+            <div class="work-right">
+                <img class="arrow-image right-arrow" src="images/arrow.png" alt="arrow-down">
+            </div>
+        </div>
+        <section class="container-4">
+            <h1 class="title-1">
+                my recent work
+            </h1>
```
Comment (mahbubul14) [inline index.html:95]: - [ ] Your "MY RECENT WORK" is not matched with the Figma design. Please work on it. :writing_hand: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
y1 | f1

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @braincee ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "Fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices, ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by karvel-code — APPROVED** [review]
Hi @braincee ,

STATUS: APPROVED!! 🥳  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (braincee) [reply index.html]: Thanks very much @mahbubul14

**Comment by braincee** [comment]
Thanks very much, @mahbubul14 and @karvel-code 

**_Note: Please I have implemented all the required and optional changes._**

love-thank-you

**Pull request merged**